### DISCUSSION TITLE
Github App: repositoryEvent not triggering when editing repo (enabling/disabling 'issues' feature)

### DISCUSSION THREAD

**Discussion by dalthviz** [community/community#24837] [Apps, API and Webhooks]
<p>Hi all,</p>
<p>Currently I’m developing a Github App and I need to ensure that the issues section is always enable (the Github App creates issues to provide feedback and steps over commits). However, even with <strong><em>Administration</em></strong>  permisions (Read &amp; Write), if I disable the issues over the repo’s settings (of a repo with the app installed), no events are being trigger. Also, it seems that other settings actually trigger a <a href="https://developer.github.com/v3/activity/events/types/#repositoryevent" rel="nofollow noopener">_ <strong>repository</strong> _</a> event. For example when a repo changes from public to private or vice versa (the payload  <strong><em>action</em></strong> of the event equals <strong><em>privatized</em></strong> or <strong><em>publicized</em></strong> ).</p>
<p>So, are there some other permissions that I need to give to my app? or maybe no events are being triggered for changes related with the <strong><em>features</em></strong> section of the repo settings? (my initial hope was that a _ <strong>repository</strong> <em> event with an <strong><em>action</em></strong> equal to </em> <strong>edited</strong> _ will be trigger for such changes over the repo settings).</p>
<p>Any guidance over this is greatly appreciated :slight_smile:</p>

**Comment by BelHFL** [comment, marked as answer]
<p>Hi <a class="mention" href="/dalthviz">@dalthviz</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by dalthviz** [comment]
<p>Thanks <a class="mention" href="/belhfl">@belhfl</a> !</p>
